metrics.md

```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
     http://www.apache.org/licenses/LICENSE-2.0
 -->
 
-## Metrics
+# Metrics
 
-Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega streams.
+Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega Streams.
 
-### Reader Metrics
+## Reader Metrics
 
-The following metrics are exposed and made available as Flink metrics for `FlinkPravegaReader` related operations.
-
-- reader group name
-- scope
-- stream names
-- online readers
-- segment positions
-- unread bytes
+The following metrics are exposed and made available as Flink metrics for `FlinkPravegaReader` related operations:
 
 Metric Name                |Description|
 |-----------------|-----------------------------------------------------------------------|
-|`readerGroupName`|The name of the reader group.|
-|`scope`|The scope name of the reader group.|
-|`streams`|The streams that are part of the reader group. The stream name will be a fully qualified name i.e., `scope/stream`|
+|`readerGroupName`|The name of the ReaderGroup.|
+|`scope`|The scope name of the ReaderGroup.|
+|`streams`|The streams that are part of the ReaderGroup. The stream name will be a fully qualified name i.e., `scope/stream`|

```

Comment:
|`streams`|The streams that are part of the `ReaderGroup` will be listed as comma separated values. The stream name will be a fully qualified name i.e., `scope/stream`|

Reply:
Accepted suggestion